### ISSUE TITLE
Vulnerability in inflight

### ISSUE THREAD

**Issue by AndreyKovanov** [rmosolgo/graphql-ruby#4859]
**Describe the bug**

Javascript client depends on inflight and it is vulnerable.
https://security.snyk.io/vuln/SNYK-JS-INFLIGHT-6095116

```
 graphql-ruby-client@1.13.1
  └─┬ glob@7.2.3
    └── inflight@1.0.6
```

**Versions**

`graphql-ruby-client` version: 1.13.1

**Steps to reproduce**

```
npm ls inflight --omit=dev
```

**Expected behavior**

`graphql-ruby-client` package doesn't depend on `inflight` package

**Actual behavior**

`graphql-ruby-client` package depends on `inflight` package

**Additional context**

To fix the vulnerability need to update `glob` package to version 10+

**Comment by rmosolgo** [comment]
Hi! Thanks for reporting this. I just released graphql-ruby-client 1.13.2 which uses Glob 10+. Let me know if you see any more trouble with it!

**Comment by AndreyKovanov** [comment]
Thank you for such a prompt reaction! It looks good now 👍🏻
